rust/examples/classification-example/src/main.rs

```diff
@@ -83,6 +85,27 @@ fn sort_results(buffer: &[f32]) -> Vec<InferenceResult> {
     results
 }
 
+fn image_to_tensor(path: String, height: u32, width: u32) -> Vec<u8> {
+    let pixels = Reader::open(path).unwrap().decode().unwrap();
+    let dyn_img: DynamicImage = pixels.resize_exact(width, height, image::imageops::Triangle);
+    let bgr_img = dyn_img.to_bgr8();
+    // Get an array of the pixel values
+    let raw_u8_arr: &[u8] = &bgr_img.as_raw()[..];
+    // Create an array to hold the f32 value of those pixels
+    let bytes_required = raw_u8_arr.len() * 4;
+    let mut u8_f32_arr:Vec<u8> = vec![0; bytes_required];
+
+    for i in 0..raw_u8_arr.len()  {
+        // Read the number as a f32 and break it into u8 bytes
+        let u8_f32: f32 = raw_u8_arr[i] as f32;
+        let u8_bytes = u8_f32.to_ne_bytes();
```

Comment:
With all of this re-arranging of bytes, I think we should be checking the top classification results in the `main` function above to make sure the classification is still correct.

Reply:
As in comparing to the original pre-tensored images? I could add .bgr versions of each of the .jpg to check that they yield the same result.

Reply:
I guess that could work, but I meant that you could figure out the top result for each of the .bgr images, put those in an array, and then in this change make sure that the .jpg images are classified with those results.